src/jit/codegenarm64.cpp

```diff
@@ -5792,7 +5965,8 @@ void CodeGen::genIntToIntCast(GenTreePtr treeNode)
         {
             // We only need to check for a negative value in sourceReg
             emit->emitIns_R_I(INS_cmp, cmpSize, sourceReg, 0);
-            genJumpToThrowHlpBlk(EJ_jl, SCK_OVERFLOW);
+            emitJumpKind jmpLTS = genJumpKindForOper(GT_LT, CK_SIGNED);
```

Comment:
Nit: at first glance I read `jmpLTS` as "jump if less than or same". Maybe `jmpLT` would be better here?